File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/MailBlockRspamd.js
goog.provide('API.Client.MailBlockRspamd');

/**
 * This is a block entry from the rspamd block list.
 * @record
 */
API.Client.MailBlockRspamd = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.MailBlockRspamd.prototype.from;

/**
 * @type {!string}
 * @export
 */
API.Client.MailBlockRspamd.prototype.subject;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/MailBlockRspamd.scala
package io.swagger.server.model


/**
 * = MailBlockRspamd =
 *
 * This is a block entry from the rspamd block list.
 *
 * @param from 
 * @param subject 
 */
case class MailBlockRspamd (
  from: String,
  subject: String
)

